### DISCUSSION TITLE
Wrong Speaker attribution

### DISCUSSION THREAD

**Discussion by djohnsonbhrp** [deepgram/community#587] [General help] [answered]
### Which Deepgram product are you using?

Deepgram API

### Details

Anything to be done about misattribution of the speaker when diarization is turned on? Here's an example of the raw text output:

"Did he ever call you to talk about it? No. Okay. And is there anything else that we haven't covered that you think would be, helpful to know?"

The single word "No" is attributed to speaker 0, but it's actually the response from speaker 1. There's maybe 1 second between when the first questions ends, then speaker 1 says, "No." and then maybe another 1 second before speaker 0 moves on to the next question. 

For the Deepgram member that's going to read this, the audio time for that paragraph starts at 14:39.

Is there anything that could be done here to improve the results?

### If you are making a request to the Deepgram API and have a request ID, please paste it below:

d52c5bc4-b230-418f-8634-7404910df0dd

**Comment by team-deepgram** [comment]
Thanks for asking your question about Deepgram! If you didn't already include it in your post, please be sure to add as much detail as possible so we can assist you efficiently, such as:
- The `request_id` if you have a question about your requests or transcription responses.
- The features you used or the full `api.deepgram.com` URL you sent your request to, including parameters.
- Any code snippets you can share.

**Comment by jpvajda** [comment, marked as answer]
@djohnsonbhrp 👋   This can happen when there is a short gap between speakers or when Deepgram misinterprets the speaker change in the conversation. 

I'm curious if can you provide the API equest you made as a `Curl` example? 

I'm curious:

* Was this streaming or pre-recorded audio?
* Which model did you used
* What other API features you might have used as that can sometimes impact the diarize output results.
